### ISSUE TITLE
NFS not working on Arch Linux with encrypted home directory [FreeBSD]

### ISSUE THREAD

**Issue by gfrey** [nathany/vagrant-gopher#6]
My home directory is encrypted. I guess this is the reason why exportfs says
  exportfs: $GOPATH/github.com/nathany/vagrant-gopher does not support NFS export

**Comment by nathany** [comment]
That could be it. It prompts for the host (Arch) password and after that gives this error?

I'm probably not going to be able to reproduce this anytime soon, but good to have it logged here. Thanks.

**Comment by gfrey** [comment]
Yes, the $GOPATH is added to `/etc/exports` but the exportfs call fails with the error mentioned.
